src/components/Post/GovernanceSideBar/index.tsx

```diff
@@ -698,8 +700,13 @@ const GovernanceSideBar: FC<IGovernanceSidebarProps> = (props) => {
 									</GovSidebarCard>
 									:
 									<GovSidebarCard className='flex items-center'>
-										You Voted: { lastVote == 'aye' ? <LikeFilled className='text-aye_green ml-2' /> : <DislikeFilled className='text-nay_red ml-2' /> }
+
+										You Voted: { lastVote == 'aye' ? <LikeFilled className='text-aye_green ml-2' /> : ''}
```

Comment:
instead of ternary operator and empty string use '&&'

Reply:
Will change it

Reply:
changed